mantidproject/vesuvio
# Use Minus() to subtract masses from total ncp

**Description of work:**
Using Minus algorithm makes sure that workspaces are compatible with each other, which is important for scientists when sometimes they have to subtract one workspace from another.

It also improves readability of the code.



**To test:**
Code review.

Manual testing:
Activate an environment with mantidworkbench installed:
  - Checkout this branch and install with `pip install .`
  - run `mvesuvio config`
  - Open Mantid workbench and open file `~/.mvesuvio/analysis_inputs.py`
  - Click Run
  - Plot workspace `analysis_inputs_fwd_0_total_ncp_sum`
  - Drag `analysis_inputs_fwd_0_1.0079_ncp_sum` onto open plot
  - Drag `analysis_inputs_fwd_0_total_ncp_except_lightest_sum` onto open plot
  - Looking at the plot, check that total ncp sum matches the sum of the other two profiles
